Sources/SwifterSwift/SwiftStdlib/DefaultStringInterpolationExtensions.swift

```diff
@@ -0,0 +1,46 @@
+// DefaultStringInterpolationExtensions.swift - Copyright 2023 SwifterSwift
```

Comment:
Since there is a new type extensions can you add it to the readme?

Reply:
Updated.